### PR TITLE
Use IG pointed to by ig.ini for configuration

### PR DESCRIPTION
**Pull request by jafeltra**
This PR addresses [CIMPL-584](https://standardhealthrecord.atlassian.net/browse/CIMPL-584).

If there is an `ig.ini` file with an `ig` property, it will be used to identify the correct ImplementationGuide to be used for the Configuration and Instance processors. The approach for finding and parsing the `ig.ini` is similar to the one used by SUSHI.

### CODE REVIEW

File: src/processor/FHIRProcessor.ts [hunk 1]
```diff
@@ -23,8 +27,11 @@ export class FHIRProcessor {
   process(): Package {
     const resources = new Package();
     let config: ExportableConfiguration;
+    const igForConfig =
+      this.lake.getAllImplementationGuides().find(doc => doc.path.endsWith(this.igPath)) ??
```
Comment (ngfreiter) [inline src/processor/FHIRProcessor.ts:31]: I don't think `endsWith` works completely here. To test this out, I had a directory with an `ig.ini` contained in a folder, `test/ig.ini`, and then had that `ig.ini` point at an IG file at the same level as the `test` folder. So the path in my `ig.ini` was something like: `..\\ImplementationGuide-hl7.fhir.us.core.json`. But the path in goFSH to the ImplementationGuide file was just `ImplementationGuide-hl7.fhir.us.core.json`, which does not end with `..\\ImplementationGuide-hl7.fhir.us.core.json`, so the ImplemetationGuide is not found.

**Review by ngfreiter — CHANGES_REQUESTED** [review]
I noticed a bit of a bug under certain circumstances.

File: package.json [hunk 2]
```diff
@@ -51,11 +51,13 @@
     "typescript": "^3.8.3"
   },
   "dependencies": {
+    "@types/ini": "^1.3.30",
```
[the remark this answers is not in the document]
Comment (jafeltra) [reply package.json:54]: Oops yes!

File: src/processor/FHIRProcessor.ts (hunk 1 above)
Comment (jafeltra) [reply src/processor/FHIRProcessor.ts]: Hm I actually don't think I knew this was allowed, which is probably why I didn't test it. I'm on it!

**Comment by jafeltra** [comment]
@ngfreiter these should be resolved in 8790f38 and afdb5e5!

**Review by ngfreiter — APPROVED** [review]
This looks good to me now!

**Review by julianxcarter — APPROVED** [review]
This looks good to me!

**Pull request merged**